src/sentry/workflow_engine/endpoints/validators/base/workflow.py

```diff
@@ -58,8 +54,8 @@ def validate_action_filters(self, value):
         return value
 
     def create(self, validated_value: dict[str, Any]) -> Workflow:
-        condition_group_validator = BaseDataConditionGroupValidator()
-        action_validator = BaseActionValidator()
+        condition_group_validator = BaseDataConditionGroupValidator(context=self.context)
+        action_validator = BaseActionValidator(context=self.context)
```

Comment:
do you need this here? it seems like it's recreated and used in the for loop over `action_filters`

Reply:
ah, actually i don't want the one in the loop (that was from a different thing i was also trying to refactor 😅)